Artifacts.toml

```
@@ -1,10 +1,10 @@
 [QSMDB]
-git-tree-sha1 = "5c82e5ff3aa99f2aa660c9ffb3caa57ee535cc21"
+git-tree-sha1 = "930fd374bd91a4b19d7adbf0cd4b5517c31b5ab2"
 lazy = true
 
      [[QSMDB.download]]
```

I am not sure if this name now needs to change maybe?

---

```suggestion
     [[QSMDB-2.download]]
```
Maybe this? but I am not sure, I would have to experiment as well

---

No, the name of the artifact should stay unless you want to change all occurrences...